### PR TITLE
chore: restrict unnecessary files being published

### PR DESCRIPTION
**Pull request by PlayMa256**
https://unpkg.com/mobx-activator@0.5.1/

too many files being published, we just need lib and few others!

### CODE REVIEW

**Comment by danfma-daitan** [comment]
Hey, @PlayMa256 !

If we remove the `lib`, which is where the built files are stored, we won't be able to use that later.

File: package.json [hunk 1]
```diff
@@ -8,6 +8,11 @@
     "build": "yarn tsc -p tsconfig.build.json",
     "test": "yarn jest"
   },
+  "files": [
```
Comment (PlayMa256) [inline package.json:11]: This means the files being published, we won't remove lib from the final tarball

Comment (danfma-daitan) [reply package.json:11]: Ops! You are right!

**Review by danfma-daitan — APPROVED** [review]

**Review by danfma-daitan — APPROVED** [review]

**Review by danfma — APPROVED** [review]
Approving with the right account... :)

**Pull request merged**